armada/armada.py

```diff
@@ -24,16 +24,14 @@ def __init__(self, args):
 
         self.tiller = Tiller()
 
-
     def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, version, chart, values in known_releases:
+        for chart_name, _, chart, values in known_releases:
```

Comment:
Changing the variable name is indeed a code change, and changes intent here.  I suggest we keep version in place currently unless it's actively breaking things.